src/dotnet-install.sh

```diff
@@ -933,8 +939,9 @@ get_http_header_wget() {
     wget $wget_options $wget_options_extra "$remote_path_with_credential" 2>&1
     wget_result=$?
 
-    if [[ $wget_result == 2 ]]; then
+    if [[ $wget_result == 1 ]] || [[ $wget_result == 2 ]]; then
         # Parsing of the command has failed. Exclude potentially unrecognized options and retry.
+        say_verbose "wget parsing failed. trying again with fewer options.."
         wget $wget_options "$remote_path_with_credential" 2>&1
         return $?
     fi
```

Comment:
Now we have a better understanding of what arguments we can pass to the command at runtime.
I suggest we remove the retries.

Reply:
done